src/lib/collab/channel.js

```diff
@@ -1,76 +1,180 @@
 import { EventEmitter2 } from 'eventemitter2'
 import { getVersion, sendableSteps } from 'prosemirror-collab'
 
+const minimumBackoff = 128 // 128ms
+const maximumBackoff = 1000 * 60 * 5 // Max 5 minutes
+const failuresBeforeCatchup = 4
+
 export class Channel {
   constructor(config, serviceClient) {
     this.config = config
     this.service = serviceClient
     this.eventEmitter = new EventEmitter2()
+    this.resetBackoff()
+    this.initializeStepsQueue()
+    this.isSending = undefined
```

Comment:
Since this var starts with an `is` I expect that the value is `true` or `false` but not undefined

Reply:
I was playing with the face that undefined is indeed a false value but still not a value defined by the code below.

Agree that this difference is useless and may introduce questions. Will fix.